src/turnkeyml/common/filesystem.py

```diff
@@ -322,10 +322,10 @@ class Keys:
     # ONNX model input tensor dimensions
     ONNX_INPUT_DIMENSIONS = "onnx_input_dimensions"
     # List of all build stages in the Sequence
-    ALL_BUILD_STAGES = "all_build_stages"
+    ALL_BUILD_STAGES = "selected_stages"
```

Comment:
I wonder if "stages_order" is a better name here as the order is the only information that we don't have in the other columns of the report

Reply:
Would "sequence" or "sequence_stages" be a better name? Sequence implies ordering :)

Reply:
Good suggestion! Maybe combine those two to get something verbose yet clear?
`selected_sequence_of_stages`

Reply:
Changed to `selected_sequence_of_stages`